### DISCUSSION TITLE
2022 Bering Sea hindcast/bridging

### DISCUSSION THREAD

**Discussion by Jason-Conner-NOAA** [afsc-gap-products/model-based-indices#42] [Team Posts]
Hi all,

I pushed my bridging exercise for Pcod to:
https://github.com/coleary-noaa/model-based-indices

Perhaps we should move this repo here when @coleary-noaa gets back from sea?

**Comment by Lewis-Barnett-NOAA** [comment]
Thanks Jason.  Indeed, we should migrate the repo to the organization and
make sure that it is accessible to all involved.  I was looking at the repo
today, but I don't have permissions to do any of these actions.  Cecilia
will be back on the ~21st though.

On Thu, Mar 17, 2022 at 9:57 AM Jason Conner ***@***.***>
wrote:

> Hi all,
>
> I pushed my bridging exercise for Pcod to:
> https://github.com/coleary-noaa/model-based-indices
>
> Perhaps we should move this repo here when @coleary-noaa
> <https://github.com/coleary-noaa> gets back from sea?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/afsc-gap-products/teams/modsquad/discussions/2>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKMJP4ITABBCXFCRLXVNMLVANP5XANCNFSM5Q7NUXRA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Lewis Barnett, PhD (he/him/his)
Research Fish Biologist

NOAA Fisheries, Alaska Fisheries Science Center
7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115

**Comment by coleary-noaa** [comment]
I would like to follow any best coding review practices we can, and now that I've reviewed as much of the code as I can have someone go through and check for quality (e.g., remove hard coded paths, local calls) before we migrate to the main page. We also never completed the discussion of how public this code is/what parts are public, which I also think we should do before we migrate.
